app/models/manageiq/providers/embedded_terraform/automation_manager/job.rb

```diff
@@ -40,15 +40,35 @@ def poll_runner
   end
 
   def post_execute
-    cleanup_git_repository
-
-    return queue_signal(:finish, message, status) if success?
+    if success?
```

Comment:
@jaisejose1123 is it possible to get logs on failure also?  I think that's almost more important in order to figure out what went wrong.